lib/request-fragment.js

```diff
@@ -19,8 +19,8 @@ const requiredHeaders = {
  * @param {Object} request - HTTP request stream
  * @returns {Promise} Response from the fragment server
  */
-module.exports = function requestFragment (fragmentUrl, fragmentAttributes, request) {
-    return new Promise((resolve, reject) => {
+module.exports = (fragmentUrl, fragmentAttributes, request) =>
```

Comment:
Irrespective of the syntax of the function, if its the `default` module function the error thrown is the same for me and defined by the file and line number. E.g. on throwing error in `request-fragment.js`, the below error is thrown with both the syntaxes:

```
if (response.statusCode >= 500) {
    reject(new Error('Internal Server Error'));
} else {
    throw new Error('test the stack trace');
    resolve(response);
}
```

> Uncaught Error: test the stack trace
      at OverriddenClientRequest.fragmentRequest.on (lib/request-fragment.js:43:23)
      at respond (node_modules/nock/lib/request_overrider.js:441:17)
      at node_modules/nock/lib/request_overrider.js:479:11
      at _combinedTickCallback (internal/process/next_tick.js:67:7)
      at process._tickCallback (internal/process/next_tick.js:98:9)